closed/src/java.base/share/classes/com/sun/crypto/provider/NativeCipherBlockChaining.java

```diff
@@ -260,6 +260,17 @@ int encrypt(byte[] plain, int plainOffset, int plainLen,
             throw new ProviderException("Internal error in input buffering");
         }
 
+        /**
+         * OpenSSL doesn't support overlapping buffers.
+         * If plaintext and ciphertext byte arrays are overlapping, make a copy of plain.
```

Comment:
This comment is out-of-date now.

Reply:
updated

Reply:
Thanks. In the future, please don't use javadoc-style comments in places like this.